website/templates/project/settings.mako

```diff
@@ -306,8 +306,8 @@
                     </div>
                     <div class="panel-body">
                         <div class="help-block">
-                            Projects can be affiliated with institutions that have created OSF for Institution accounts.
                             <!-- ko if: affiliatedInstitutions().length == 0 -->
+                            Projects can be affiliated with Institutions that have created OSF for Institution accounts.
```

Comment:
I thought that was the typo.  What is the typo from https://github.com/CenterForOpenScience/osf.io/pull/5726#discussion-diff-64600674